autotest/pytest.ini

```diff
@@ -1,6 +1,14 @@
 [pytest]
 python_files = *.py
 testpaths = ogr gcore gdrivers osr alg gnm utilities pyscripts
+log_file = autotest.log
+log_file_level = INFO
+env =
+  GDAL_DOWNLOAD_TEST_DATA=NO
+  GDAL_RUN_SLOW_TESTS=NO
+  LD_LIBRARY_PATH=../gdal
+  GDAL_DATA=../gdal/data
+#  GDAL_DRIVER_PATH=/usr/lib/gdal/gdalplugins
```

Comment:
There would be a discussion what is default here or just add it for help as commented out.

Reply:
These are examples so commented out.